src/pages/fieldManagement/settings/defaultFieldComponents/SpeciesEditor.jsx

```diff
@@ -151,7 +178,10 @@ export default function SpeciesEditor({
             style={{ width: '100%' }}
           />
           <Button
-            onClick={() => setSearchTerm(searchInput)}
+            onClick={() => {
+              setError(null);
+              setSearchTerm(searchInput);
+            }}
```

Comment:
Because this is the submission button for a form, `setError(null)` can be moved to the form's `onSubmit`.